test/signalrclienttests/connection_impl_tests.cpp

```diff
@@ -1612,3 +1606,105 @@ TEST(connection_impl_reconnect, can_stop_connection_from_reconnecting_event)
     ASSERT_EQ(_XPLATSTR("[state change] reconnecting -> disconnecting\n"), remove_date_from_log_entry(log_entries[3]));
     ASSERT_EQ(_XPLATSTR("[state change] disconnecting -> disconnected\n"), remove_date_from_log_entry(log_entries[4]));
 }
+
+
+TEST(connection_impl_reconnect, test_can_stop_connection_from_reconnecting_event)

```

Comment:
fix the name to hide the fact that this test was copy-pasted and then tweaked...